packages/tools/webpack-fluid-loader/src/routes.ts

```diff
@@ -287,7 +287,6 @@ const fluid = (req: express.Request, res: express.Response, baseDir: string, opt
             options,
             document.getElementById("content"))
         .then(() => fluidStarted = true)
-        .catch((error) => console.error(error));
```

Comment:
this just makes debugging harder, as i can't just break on all uncaught errors, the errors log regardless